pkg/apis/triggers/v1alpha1/trigger_validation_test.go

```diff
@@ -165,6 +187,9 @@ func Test_TriggerValidate(t *testing.T) {
 }
 
 func TestTriggerValidate_error(t *testing.T) {
+	var simpleResourceTemplate = runtime.RawExtension{
+		Raw: []byte(`{"kind":"PipelineRun","apiVersion":"tekton.dev/v1alpha1","metadata":{},"spec":{},"status":{}}`),
+	}
```

Comment:
nit: This can be brittle if we need to modify this . Let's break start with a PipelineRun type and define a func that will return the RawExtension?

e.g.

```go
func rawExtension(t *testing.T, a interface{}) runtime.RawExtension {
  t.Helper()
  o, err := json.Marshal(a)
  if err != nil {
    t.Fatal(err)
  }
  return runtime.RawExtension{Raw: b}
}
```

Then we can probably also get rid of this top level var and just inline it below (since this is only used once)

Reply:
neat, yeah the string approach is pretty brittle

Reply:
I changed it here and opened #796 for cleaning up other usages.